be/src/runtime/memory/mem_tracker_limiter.cpp

```diff
@@ -229,6 +229,7 @@ std::string MemTrackerLimiter::log_process_usage_str() {
     std::vector<MemTracker::Snapshot> snapshots;
     MemTrackerLimiter::make_process_snapshots(&snapshots);
     MemTrackerLimiter::make_type_snapshots(&snapshots, MemTrackerLimiter::Type::GLOBAL);
+    MemTrackerLimiter::make_type_snapshots(&snapshots, MemTrackerLimiter::Type::COMPACTION);
```

Comment:
Only print `Label=BaseCompaction`, do not print its child tracker, otherwise it may print a lot

Reply:
can add a parameter like this
![efd350c1-640b-4ea2-8a45-f609442fa621](https://github.com/apache/doris/assets/13197424/67c5ebae-f0f6-46fb-aef2-21d5e46a48eb)
Then here make_type_snapshots(COMPACTION, with_child=false)

Reply:
Can be added at the same time
`MemTrackerLimiter::make_type_snapshots(&snapshots, MemTrackerLimiter::Type::QUERY, false);`
`MemTrackerLimiter::make_type_snapshots(&snapshots, MemTrackerLimiter::Type::LOAD, false);`
`MemTrackerLimiter::make_type_snapshots(&snapshots, MemTrackerLimiter::Type::COMPACTION, false);`